src/test/java/net/snowflake/client/jdbc/CustomProxyIT.java

```diff
@@ -1,40 +1,45 @@
 package net.snowflake.client.jdbc;
 
 import static junit.framework.TestCase.assertEquals;
-import static junit.framework.TestCase.assertTrue;
 import static junit.framework.TestCase.fail;
 import static net.snowflake.client.AbstractDriverIT.getFullPathFileInResource;
+import static net.snowflake.client.jdbc.SnowflakeDriverIT.findFile;
 import static net.snowflake.client.jdbc.SnowflakeUtil.systemGetProperty;
 import static org.hamcrest.CoreMatchers.equalTo;
 import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.Assert.assertTrue;
 
+import java.io.File;
 import java.net.Authenticator;
 import java.net.PasswordAuthentication;
-import java.net.URL;
-import java.sql.Connection;
-import java.sql.DriverManager;
-import java.sql.ResultSet;
-import java.sql.SQLException;
-import java.sql.Statement;
+import java.sql.*;
 import java.util.Properties;
 import net.snowflake.client.category.TestCategoryOthers;
 import net.snowflake.common.core.SqlState;
 import org.junit.Ignore;
+import org.junit.Rule;
 import org.junit.Test;
 import org.junit.experimental.categories.Category;
+import org.junit.rules.TemporaryFolder;
+
+// To run these tests, you must:
+// 1.) Start up a proxy connection. The simplest ways are via Squid or BurpSuite
+// 2.) Enter your own username and password for the account you're connecting to
+// 3.) Adjust parameters like role, database, schema, etc to match with account accordingly
 
 @Category(TestCategoryOthers.class)
 public class CustomProxyIT {
+  @Rule public TemporaryFolder tmpFolder = new TemporaryFolder();
+
   @Test
   @Ignore
   public void testCorrectProxySettingFromConnectionString()
       throws ClassNotFoundException, SQLException {
 
     String connectionUrl =
         "jdbc:snowflake://s3testaccount.us-east-1.snowflakecomputing.com/?tracing=ALL"
-            + "&proxyHost=localhost&proxyPort=3128"
-            + "&proxyUser=testuser1&proxyPassword=test"
-            + "&nonProxyHosts=*.foo.com%7Clocalhost&useProxy=true";
+            + "&proxyHost=localhost&proxyPort=8080"
+            + "&useProxy=true";
```

Comment:
Just to double check, this settings equals no proxy, right? And we are only adding this to pretending we are using a proxy

Reply:
These settings do use a proxy. Setting useProxy=true will cause the proxy to be used.

Reply:
Oh yeah yeah, I was trying to say the proxy is itself (localhost)